website/content/docs/nextjs.md

```diff
@@ -152,7 +152,7 @@ Paste HTML for Netlify CMS into your `public/admin/index.html` file (check out t
 </head>
 <body>
   <!-- Include the script that builds the page and powers Netlify CMS -->
-  <script src="https://unpkg.com/netlify-cms@2.9.7/dist/netlify-cms.js"></script>
+  <script src="https://unpkg.com/netlify-cms@^2.10.47/dist/netlify-cms.js"></script>
```

Comment:
See [this thread](https://community.netlify.com/t/large-media-storage/12915/16). Large Media was broken before a bugfix pushed in v2.10.47.